src/events/mouse.js

```diff
@@ -929,4 +990,86 @@ p5.prototype._onwheel = function(e) {
   }
 };
 
+/**
+ * <p>The function <a href="#/p5/requestPointerLock">requestPointerLock()</a>
+ * locks the pointer to its current position and makes it invisible.
+ * Use <a href="#/p5/movedX">movedX</a> and <a href="#/p5/movedY">movedY</a> to get the difference the mouse was moved since
+ * the last call of draw</p>
+ * <p>Note that not all browsers support this feature</p>
+ * <p>This enables you to create experiences that aren't limited by the mouse moving out of the screen
+ * even if it is repeatedly moved into one direction. </p>
+ * <p>For example a first person perspective experience</p>
+ *
+ * @method requestPointerLock
+ * @example
+ * <div class="notest">
+ * <code>
+ * let cam;
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ *   requestPointerLock();
+ *   cam = createCamera();
+ * }
+ *
+ * function draw() {
+ *   background(255);
+ *   cam.pan(-movedX * 0.001);
+ *   cam.tilt(movedY * 0.001);
+ *   sphere(25);
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * black 50x50 rect moves up and down with mouse move while no cursor is visible
+ *
+ */
+p5.prototype.requestPointerLock = function() {
+  // pointer lock object forking for cross browser
+  var canvas = this._curElement.elt;
+  canvas.requestPointerLock =
+    canvas.requestPointerLock || canvas.mozRequestPointerLock;
+  if (!canvas.requestPointerLock) {
+    console.log('requestPointerLock is not implemented in this browser');
+    return false;
+  }
+  canvas.requestPointerLock();
+  return true;
+};
+
+/**
+ * <p>The function <a href="#/p5/exitPointerLock">exitPointerLock()</a>
+ * exits a previously triggered <a href="#/p5/requestPointerLock">pointer Lock</a>
+ * for example to make ui elements usable etc
+ *
+ * @method exitPointerLock
+ * @example
+ * <div class="notest">
+ * <code>
+ * //click the canvas to lock the pointer
+ * //click again to exit (otherwise escape)
+ * let locked = false;
+ * function draw() {
+ *   background(237, 34, 93);
+ * }
+ * function mouseClicked() {
+ *   if (!locked) {
+ *     locked = true;
+ *     requestPointerLock();
+ *   } else {
+ *     exitPointerLock();
+ *     locked = false;
+ *   }
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * black 50x50 rect moves up and down with mouse move while no cursor is visible
```

Comment:
Alright, truly almost done, can you just make these bits of alt text correct again? And make sure there is alt text for every example. Thanks!

Reply:
done